main/src/main/java/org/mobilitydata/gtfsvalidator/runner/ValidationRunner.java

```diff
@@ -229,18 +233,31 @@ public static void exportReport(
             "Error creating output directory: %s", config.outputDirectory());
       }
     }
+    SimpleDateFormat formatter = new SimpleDateFormat("yyyy-MM-dd 'at' HH:mm:ss z");
```

Comment:
I see here you are using SimpleDateFormat. I recommend DateTimeFormatter. DateTimeFormatter is a replacement for the old SimpleDateFormat that is thread-safe and provides additional functionality. I recommend that the new Date Time API introduced in Java 8.

Reply:
This piece of code already existed and was simply moved from main/src/main/java/org/mobilitydata/gtfsvalidator/report/HtmlReportGenerator.java.
I would prefer at this point to keep it as is.

Reply:
I see that the GtfsDate is already using LocalDate which is introduced with DateTimeFormatter in the new java 8 date time API. I'm ok with keeping it for now for this issue, but shall we open an issue about updating to the java 8 data time API since it's more immutable and threadsafe than the old ones? @davidgamez

Reply:
I was willing to give it a go, but DateTimeFormatter had trouble interpreting the part of the pattern related to time zone. 
Since like I said the date related code in this PR is actually some code that was moved and not modified, I prefer to keep it the way it is.

Reply:
In reality I don't think you should open an issue about this.
The fact that it's thread safe is irrelevant here because we are creating and using the formatter in the same thread. As for extra functionalities, we don't need them here.